module/actor.js

```diff
@@ -1014,6 +1025,24 @@ class DCCActor extends Actor {
       })
     }
 
+    // Add Strength or Agility modifier to attack rolls
+    let modifier;
+    let modifierLabel;
+    if ((this.system.class.className) && (game.settings.get('dcc', 'automateCombatModifier'))) {
```

Comment:
Is there a reason for checking that className is not a falsey value here?

Reply:
It s not needed - removed.